app/routes/image-set/lion-search.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import OrganizationRouteMixin from 'lion-guardians/mixins/organization-route';
+
+export default Ember.Route.extend(OrganizationRouteMixin, {
+  model: function() {
+    return [];
```

Comment:
I was wondering for a while why there is an empty array here until I realized the that the controller will populate an array for the `model` property in its `displayResults` action — right? A comment or explanatory variable name would help.

And this is a cognitive mismatch with the place where we set model to `null` in `resetController`, below. It should also set to an empty array if that is the `reset`/default state for this route.